src/graphql/index.ts

```diff
@@ -0,0 +1,25 @@
+import { gql, Config } from "apollo-server";
+import { DocumentNode } from "graphql";
+import { registerNewCart } from "../cart/cart";
+import { productsResolvers } from "./product/product.resolver";
+import { productsSchema } from "./product/product.schema";
+import { ProductTypeDef } from "./product/product.types.schema";
+
+const baseSchema: DocumentNode = gql`
```

Comment:
I don't think you need this

Reply:
In this project u right but if i want to extend it I have to extend the base query